src/redux/Actions/ActionTypes/DashBoardActions.js

```diff
@@ -0,0 +1,25 @@
+import * as types from '../index';
```

Comment:
Just rename all the index.js files to meaningful names. So that by seeing the import we can understand the content/overall functionality of the file